gulpfile.js

```diff
@@ -127,7 +127,9 @@ function createWebpackConfig(defines, output) {
     BUNDLE_VERSION: versionInfo.version,
     BUNDLE_BUILD: versionInfo.commit,
   });
-  var licenseHeader = fs.readFileSync('./src/license_header.js').toString();
+  var licenseHeaderLibre = fs
+                           .readFileSync('./src/license_header_libre.js')
+                           .toString();
```

Comment:
This isn't how we usually break lines in PDF.js, let's use to following format instead:
```javascript
  var licenseHeaderLibre =
    fs.readFileSync('./src/license_header_libre.js').toString();
```

Reply:
Yup , I get it . Working on the indentation now